src/main/kotlin/org/opensearch/integrations/model/IntegrationObjectDataProperties.kt

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.integrations.model
+
+import org.opensearch.common.io.stream.Writeable
+import org.opensearch.core.xcontent.XContentParser
+
+internal object IntegrationObjectDataProperties {
```

Comment:
missing the integration actual fields - see [here](https://github.com/opensearch-project/observability/blob/main/src/main/resources/schema/system/integration.schema)